plugins/anomaly-detection/anomaly_detection.py

```diff
@@ -416,23 +533,43 @@ def anomaly_detection(algo_type, db_jdbc_address, start_time, end_time,
             "password", os.getenv("CH_PASSWORD")).option(
             "query", sql_query).load()
     )
+    group_columns = DF_AGG_GRP_COLUMNS if agg_flow else DF_GROUP_COLUMNS
 
-    prepared_DF = initDF.groupby(DF_GROUP_COLUMNS).agg(
+    prepared_DF = initDF.groupby(group_columns).agg(
         f.collect_list("flowEndSeconds").alias("flowEndSeconds"),
         f.stddev_samp("max(throughput)").alias("throughputStandardDeviation"),
-        f.collect_list(f.struct(["Diff_Secs", "max(throughput)"])).alias(
-            "Diff_Secs, Throughput"))
+        f.collect_list(f.struct(["max(throughput)"])).alias("Throughput"))
```

Comment:
As we discussed last Friday, this should be sum(throughput) for aggregated TAD, I noticed you put that changes on https://github.com/antrea-io/theia/pull/192, can you move all functional code changes inside this PR and let #192 have test codes only?